docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md

```diff
@@ -0,0 +1,89 @@
+- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.s 
+- **Unbundled pricing (or pay-per-use pricing):** Clients not using the Expensify Card are billed at $18/seat monthly.
+- **No annual commitment:** Partners pay only for what they use, with no annual subscriptions required.
+- **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill
+
+# Understanding the billing process
+
+Expensify bills the owner of the expense policy for the activity on that policy. If accountants retain ownership of client policies, they receive the bill and can then re-bill their clients based on individual agreements.
+
+Each month, Expensify will send a consolidated bill detailing:
+- **Pay-per-use seats:** This is the number of active clients and their users for the month.
+- **Expensify Card discount**: This amount reflects how much spend is put on your card, which then determines the discount for that month.
+- **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats
+- **Policy list:** This is an overview of all client policies with their respective active seats.
+
+## Consolidated Domain Billing
+
+If your firm wishes to consolidate all Expensify billing to a single account, the Consolidated Domain Billing feature is your go-to tool. It centralizes payment for all group policies owned by any domain member of your firm.
+
+### Activating Consolidated Domain Billing:
+  1. Claim and verify your firm’s domain.
+  2. Navigate to **Settings > Domains > [Domain Name] > Domain Admins** and set a **"Primary Domain Admin"** by using the drop down toggle to select an email address. 
+  3. Enable **Consolidated Domain Billing** in the same section.
+
+The Consolidated Domain Billing tool ensures that billing takes place under a single Expensify account associated with your firm. This eliminates the need to add multiple payment cards across various accounts to cover payments for multiple clients.
+
+## Maintaining a Console of all clients:
+
+If your firm wants to have a console view of all client policies and domains, you will want to create a single, centralized login to manage all client policies and domains, such as accounting@myfirm.com. 
+
+  1. Create a dedicated email address that will act as the universal policy owner, for example, accounting@myfirm.com.
+  2. Register this email with Expensify or your chosen platform and ensure it is verified and secured.
+  3. Within each client policy settings, add your centralized email (e.g. accounting@myfirm.com) as a policy admin.
+  4. Do the same with each client domain.
+
+
```

Comment:
```suggestion
```

Reply:
it doesn't affect the way article is rendered. So it's ok to not make this change